src/colvaratoms.cpp

```diff
@@ -384,12 +388,16 @@ int cvm::atom_group::parse(std::string const &group_conf)
 
   // We need to know about fitting to decide whether the group is scalable
   // and we need to know about scalability before adding atoms
-  bool b_defined_center = get_keyval(group_conf, "centerReference", b_center, false);
-  bool b_defined_rotate = get_keyval(group_conf, "rotateReference", b_rotate, false);
+  bool b_defined_center = get_keyval_feature(this, group_conf, "centerToOrigin", f_ag_center_origin, false);
+  b_defined_center |= get_keyval_feature(this, group_conf, "centerReference", f_ag_center, is_enabled(f_ag_center_origin));
+  if (is_enabled(f_ag_center_origin) && ! is_enabled(f_ag_center)) {
```

Comment:
In here, it is assumed that `ref_pos_cog` is (0, 0, 0) by default, which is true but only because its class has a proper default constructor and `cvm::atom_group::center_ref_pos()` is never called without a set of reference positions.

I would set `ref_pos_cog` explicitly as the origin right when `f_ag_center_origin` is enabled. (Actually, this is one of the situations where a map of functions in the `colvardeps` may be useful to handle callbacks: add it to the C++11 wish list?) This way, if we add COM support instead of COG later, we could do it consistently without risking to introduce subtle bugs.